build.sh

```diff
@@ -125,6 +125,6 @@ set -x # Print commands from now on
     --disable-usb-redir \
     ${user_opts}
 
-time make -j"${job_count}" 2>&1 | tee build.log
+time make -j"${job_count}" subdir-i386-softmmu 2>&1 | tee build.log
```

Comment:
The "subdir-" make-option sounds quite aggressive and I'm not sure how it really works. I think that should have been a separate change (unless removing the option adds more dependencies to CI); if the Windows CI builds work I'm fine with trusting this.

I don't expect this to do much, as we already restrict targets to i386-softmmu anyway (and we'll disable even more things in #267).